src/collective/fingerpointing/subscribers/configure.zcml

```diff
@@ -97,4 +97,10 @@
       handler=".iterate_logger"
       />
 
+  <!-- Workflow -->
+  <subscriber
+      for="Products.CMFCore.interfaces.IActionSucceededEvent"
+      handler=".workflow_logger"
+      />
+
```

Comment:
maybe we should add **plone.app.contenttypes** to our [testing matrix](https://github.com/collective/collective.fingerpointing/blob/master/.travis.yml) to run the tests for both **Archetypes** and **Dexterity**.